web/app/model/Command.js

```diff
@@ -20,12 +20,26 @@ Ext.define('Traccar.model.Command', {
     identifier: 'negative',
 
     fields: [{
+        name: 'id',
+        type: 'int'
+    }, {
+        name: 'description',
+        type: 'string'
+    }, {
         name: 'deviceId',
         type: 'int'
     }, {
         name: 'type',
         type: 'string'
     }, {
+        name: 'name',
+        convert: function (v, rec) {
+            var name = Strings['command' + rec.get('type').charAt(0).toUpperCase() + rec.get('type').slice(1)];
```

Comment:
We have similar code in other places. Should be have some helper method to do it?